src/gerbil/runtime/system.ss

```diff
@@ -45,7 +45,7 @@ namespace: #f
   => :string
   (build-manifest-string gerbil-system-manifest))
 
-(defmutable gerbil-greeting (gerbil-system-version-string) : :string)
+(defmutable* gerbil-greeting (gerbil-system-version-string) : :string)
```

Comment:
Same question.

Need to update gerbil/boot-gxi.c, gerbil/interactive/init.ss.

Reply:
same reply.

Reply:
You still need to update gerbil/boot-gxi.c